cmd/snapshotgc/snapshotgc_test.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/go-logr/zapr"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	releasev1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	"go.uber.org/zap"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var _ = Describe("Test garbage collection for snapshots", func() {
+	var logger logr.Logger
+
+	BeforeEach(func() {
+		zapLog, err := zap.NewDevelopment()
+		if err != nil {
+			panic(fmt.Sprintf("who watches the watchmen (%v)?", err))
+		}
+		logger = zapr.NewLogger(zapLog)
```

Comment:
What is the reason behind using two different loggers? 
Can't we use just one or the another?

Reply:
For creating the logger, I followed this: https://github.com/go-logr/zapr?tab=readme-ov-file#zapr-zap

I honestly don't know why it's created this way. 

Per @MartinBasti request, I used this library as it implements the `logr` interface, that is suppose to help us keeping consistency between the script's logs and the controller's logs.